vignettes/comparison.Rmd

```diff
@@ -0,0 +1,322 @@
+---
+title: Exemplary comparison of Bayesian and frequentist MMRM fits
```

Comment:
Do you mean "Example comparison..."?

Reply:
I wanted to highlight that it's not a systematic or in any way generalizable comparison. Indeed, "exemplary" is somewhat misleading. I will change this in the text and suggest to state only "Comparison of Bayesian and frequentist MMRM fits" in the title, for simplicity. Please let me know if you have any preferred wording.